instrumentation/github.com/gocql/gocql/otelgocql/gocql_test.go

```diff
@@ -25,70 +25,25 @@ import (
 
 	"github.com/stretchr/testify/require"
 
+	"go.opentelemetry.io/otel/api/metric"
 	"go.opentelemetry.io/otel/api/trace"
 	"go.opentelemetry.io/otel/semconv"
 
-	"go.opentelemetry.io/otel/sdk/metric/controller/push"
-	"go.opentelemetry.io/otel/sdk/metric/processor/basic"
-	"go.opentelemetry.io/otel/sdk/metric/selector/simple"
-
 	"github.com/gocql/gocql"
 	"github.com/stretchr/testify/assert"
 
+	mockmeter "go.opentelemetry.io/contrib/internal/metric"
```

Comment:
I'm aware that that these test helpers are now available in the main OTEL repo (related to discussion in https://github.com/open-telemetry/opentelemetry-go-contrib/pull/190#discussion_r465431109).

I wanted to open a ticket to replace these internal helpers with the ones from the OTEL repo, however I noticed that `opentelemetry-go-contrib` has also own internal helpers for tracer, even though these existed already in the OTEL repo.

Is there a reason for this duplicity or should tracer test helpers be replaced by the helpers from OTEL repo as well?

Reply:
> I wanted to open a ticket to replace these internal helpers with the ones from the OTEL repo, however I noticed that `opentelemetry-go-contrib` has also own internal helpers for tracer, even though these existed already in the OTEL repo.
> 
> Is there a reason for this duplicity or should tracer test helpers be replaced by the helpers from OTEL repo as well?

No, we should unify on a single implementation in the `otel` repo IMO. I think the duplicity came from the concurrent development of both testing libraries more than an explicit choice. Unifying will avoid drift in implementation and avoid uncertainty by future development on which package to use.

Reply:
Got it, created https://github.com/open-telemetry/opentelemetry-go-contrib/issues/385 to track it.